tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -390,6 +436,51 @@ def _create_fsx_open_zfs_volume_ids(num_existing_fsx_open_zfs_volumes, fsx_facto
     return []
 
 
+def _get_fsx_file_cache_id(num_existing_fsx_file_caches, create_fsx_file_cache, file_cache_path, bucket_name):
+    if num_existing_fsx_file_caches > 0:
+        return create_fsx_file_cache(file_cache_path, bucket_name)
+    return []
+
+
+@pytest.fixture
+def create_fsx_file_cache(request, region, vpc_stack: CfnVpcStack, cfn_stacks_factory):
+    """Create a FSx File Cache Stack with its required Networking resources."""
+    file_cache_config_file = os.path.join("resources", "file-cache-storage-cfn.yaml")
+
+    def _create_fsx_file_cache(file_cache_path, bucket_name):
+        if request.config.getoption("external_shared_storage_stack_name"):
```

Comment:
This option currently refers to the stack deployed as part of DFSM integ tests, i.e. `/aws-parallelcluster/tests/integration-tests/tests/update/test_update/test_dynamic_file_systems_update/storage-stack.yaml`.

It's correct to support the reuse of the stack, but we should have a separated option because every option should refer to a specific stack to avoid ambiguity.

Reply:
> external_shared_storage_stack ( as nested stack) for the DFSM tests too

Did not notice that, then we are fine.